### DISCUSSION TITLE
Welcome to libre-weather Discussions!

### DISCUSSION THREAD

**Discussion by TobiTenno** [LibreWeather/libre-weather#18] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TobiTenno** [comment, marked as answer]
Is this threaded?

**Reply by TobiTenno** [reply]
This one is

**Comment by jkuester** [comment]
Wat, how did different?

**Reply by jkuester** [reply]
Y tf would the repo discussions be threaded but not team discussions?

**Reply by TobiTenno** [reply]
cause silliness
